Documentation.java

```diff
@@ -0,0 +1,42 @@
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.visitor.ModifierVisitor;
+import com.github.javaparser.ast.visitor.Visitable;
+import com.github.javaparser.utils.SourceRoot;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public  static  void main(String[] args) throws IOException {
+        Path dirPath =  Paths.get("C:\\JavaParserTest\\src\\main\\resources\\src\\");
```

Comment:
This file exists only on a specific machine, tests should not depend on where they are executed.
do not provide absolute paths to files